docs/usage/configuration-options.md

```diff
@@ -368,6 +368,12 @@ If you need to _override_ constraints that Renovate detects from the repository,
 
 Note: make sure not to mix this up with the term `compatibility`, which Renovate uses in the context of version releases, e.g. if a Docker image is `node:12.16.0-alpine` then the `-alpine` suffix represents `compatibility`.
 
+## deepExtract
+
+If configured to `true`, then dependency extraction will be done using the relevant package manager instead of JS-based parsing.
```

Comment:
```suggestion
If configured to `true`, then dependency extraction will be done using the relevant package manager instead of JavaScript-based parsing.
```